packages/meditrak-app-server/src/index.ts

```diff
@@ -8,16 +8,31 @@ import * as dotenv from 'dotenv';
 import http from 'http';
 
 import winston from 'winston';
+import { ModelRegistry, TupaiaDatabase } from '@tupaia/database';
 import { configureWinston } from '@tupaia/server-boilerplate';
+import { isFeatureEnabled } from '@tupaia/utils';
 import { createApp } from './app';
+import { MeditrakAppServerModelRegistry } from './types';
+import { ServerChangeEnqueuer } from './sync';
 
 configureWinston();
 dotenv.config(); // Load the environment variables into process.env
 
+const database = new TupaiaDatabase();
+const models = new ModelRegistry(database) as MeditrakAppServerModelRegistry;
+
 /**
  * Set up app with routes etc.
  */
-const app = createApp();
+const app = createApp(database);
+
+/**
+ * Set up change handler for the meditrakSyncQueue
+ */
+if (isFeatureEnabled('SERVER_CHANGE_ENQUEUER')) {
+  const serverChangeEnqueuer = new ServerChangeEnqueuer(models);
+  serverChangeEnqueuer.listenForChanges();
+}
```

Comment:
Running the `ServerChangeEnqueuer` here rather than on the `central-server` as previous. This means that the `meditrak-app-server` would need to be running to ensure that changes get synced to the `meditrak-app`.

Also the first time we're running a change handler outside of the `central-server`. Happy to hear reviewer's opinions on it, I can definitely see pros and cons...

Reply:
Hmmmm....yes, I can see pros and cons too! I'll think on this...

There might be some CI/CD changes to make because of this requirement btw. For e.g. I think we wait for the meditrak-server pm2 process to post that it's ready before we run migrations.

Reply:
Ah good point... I guess we could just launch the `meditrak-app-server` first? But then we're starting to wade into an uncomfortable territory where various services rely on others running prior to them... Perhaps central-server could check to see if meditrak-app-server is running before it kicks off the migrations?

Lets chat on it. I don't know if I feel too strongly either way, but keeping it here certainly keeps the code cleaner 😄